tests/full_mode/run.sh

```diff
@@ -40,8 +40,7 @@ function fail_acquire_global_lock() {
     cp $cur/conf/dm-task.yaml $WORK_DIR/dm-task.yaml
     sed -i '/timezone/i\ignore-checking-items: ["dump_privilege"]' $WORK_DIR/dm-task.yaml
     run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-        "start-task $WORK_DIR/dm-task.yaml --remove-meta" \
-        "\"result\": true" 1
+        "start-task $WORK_DIR/dm-task.yaml --remove-meta"
```

Comment:
Do `start-task` check in other cases have the same problem?

Reply:
seems other are "successful start" or "failed at init stage"